streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasks.java

```diff
@@ -510,6 +510,17 @@ void clear() {
         suspended.clear();
     }
 
+    @Override
+    public void shutdown(final boolean clean) {
+        log.debug("Shutting down all active tasks in" + "\n" +
```

Comment:
nit: `in {} mode" + ` and add clean boolean?.

Ditto otherwise.

Reply:
Ack